File: platforms/android/java/app/msime/client/JapaneseNineKeyActions.java
package app.msime.client;

/** Japanese side-key labels; composition and conversion remain owned by Engine. */
public final class JapaneseNineKeyActions {
    private JapaneseNineKeyActions() { }

    public static String spaceTitle(boolean composing) {
        return composing ? "変換" : "空白";
    }

    public static String returnTitle(boolean composing) {
        return composing ? "確定" : "改行";
    }
}

File: platforms/harmony/entry/src/main/ets/keyboard/JapaneseNineKeyActions.ts
/**
 * Japanese side-key labels, ported from
 * platforms/android/java/app/msime/client/JapaneseNineKeyActions.java.
 *
 * Composition and conversion stay owned by the Engine; these are only what the two side keys print
 * depending on whether something is being composed.
 */
export class JapaneseNineKeyActions {
  static spaceTitle(composing: boolean): string {
    return composing ? '変換' : '空白';
  }

  static returnTitle(composing: boolean): string {
    return composing ? '確定' : '改行';
  }
}
